### ISSUE TITLE
What is the source for extracted_temporal_relation.json?

### ISSUE THREAD

**Issue by achew012** [wenhycs/NAACL2021-Event-Time-Extraction-and-Propagation-via-Graph-Attention-Networks#2]
Hi, i was trying to replicate the paper outputs by mapping our data to the files in dataloader.py and was wondering how is extracted_temporal_relation.json constructed and if it is provided in the repository?

**Comment by wenhycs** [comment]
Hi, you can find the temporal relation extraction that we use at: https://github.com/qiangning/NeuralTemporalRelation-EMNLP19
The inputs are all event pairs within two-sentence (following the MATRES dataset from which the temporal relation extraction model is trained). We only keep the relations whose confidence score is over 90%.

**Comment by achew012** [comment]
Hi Haoyang, thanks for the reply. I was looking at the repo that you posted and was curious how did you convert the text into the input (e.g.  the tagging scheme and E1,E2,B,M,A labels in the sentences)? Is it possible to share the preprocessing script? Also, did you directly infer the temporal relation extraction of https://github.com/qiangning/NeuralTemporalRelation-EMNLP19 on the MATRES pretrained model or converted your labels in data.json for training on ACE05 directly? Thank you!

**Comment by wenhycs** [comment]
> I was looking at the repo that you posted and was curious how did you convert the text into the input (e.g. the tagging scheme and E1,E2,B,M,A labels in the sentences)?

The way we convert is to use all contexts within a two-sentence range, enumerate event pairs from those contexts, and assign E1/E2 based on their offset (the offsets of E1s should be always smaller than E2s). As for B/M/A labels, B label means the token is before E1 in the context, M means in the middle, and A means after. If you look into their code, you will find that those additional tags are not used in their model.

> Is it possible to share the preprocessing script?

Sorry I did not maintain the format conversion code. I might need to check back later to see if I can still find it.

> Also, did you directly infer the temporal relation extraction of https://github.com/qiangning/NeuralTemporalRelation-EMNLP19 on the MATRES pretrained model or converted your labels in data.json for training on ACE05 directly? 


We don't have temporal relation annotation on ACE05, so we just use the checkpoint from https://github.com/qiangning/NeuralTemporalRelation-EMNLP19 (which is trained on MATRES) and apply the model to ACE05 data.
